src/Controls/src/Core/Handlers/Items/iOS/StructuredItemsViewController.cs

```diff
@@ -167,6 +167,11 @@ internal void UpdateSubview(object view, DataTemplate viewTemplate, nint viewTag
 
 			if (uiView != null)
 			{
+				if (uiView.Superview is WrapperView wrapperView)
```

Comment:
I'm wondering if the uiView passed into this view is the wrong view

Line `158` calls `RemovefromSuperView`

If uIVIew has a `WrapperView` that's going to cause some weird behavior

Reply:
The uiView passed to this view is a Wrapper view, but when it goes here: UpdateView(view, viewTemplate, ref uiView, ref formsElement); Line `166` the wrapper view is replaced by the view it wraps

Reply:
Here: https://github.dev/kubaflo/maui/blob/0975e08cc5a3c78b30918e39cbe532d9df4ed173/src/Controls/src/Core/Handlers/Items/iOS/TemplateHelpers.cs#L29

Reply:
I wonder if the overall fix here would be to return "ToPlatform()" from `RealizeView` vs ".PlatformView"

Reply:
It works!!